tests/core/conversations_worker/integration/broadcast_health_check.py

```
@@ -0,0 +1,232 @@
+                subscribed.set()
+
+        await ch.subscribe(_on_sub)
+        await asyncio.wait_for(subscribed.wait(), timeout=10)
+        broadcast_ch = ch
+        log.info("Broadcast sender connected on topic=%s", topic)
+        while True:
+            await asyncio.sleep(1)
+
+    try:
+        loop.run_until_complete(_setup())
+    except BaseException as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `BaseException`

(BLE001)